generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -197,48 +192,61 @@ Scenario: Running a 'shorterThan' request alongside a contradicting longerThan (
 Scenario: Running a 'shorterThan' request alongside a non-contradicting shorterThan constraint should be successful
      Given foo is shorter than 4
        And foo is shorter than 3
-       And foo is matching regex /[♀]{0,5}/
-     Then the following data should be included in what is generated:
+       And foo is matching regex /[♀]{1,5}/
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
        | foo  |
        | null |
+       | "♀"  |
        | "♀♀" |
        And the following data should not be included in what is generated:
```

Comment:
I don't think you need to keep this section in the test as it already defines exactly what is generated given the changes you have made

Reply:
removed